CMakeLists.txt

```diff
@@ -346,6 +346,26 @@ message(STATUS "linux probes: ${ENABLE_PROBES_LINUX}")
 message(STATUS "solaris probes: ${ENABLE_PROBES_SOLARIS}")
 message(STATUS " ")
 
+
+message(STATUS "UNIX PROBES (depending of specific libs):")
+message(STATUS "Unix fileextendedattribute probe: ${OPENSCAP_PROBE_UNIX_FILEEXTENDEDATTRIBUTE}")
+message(STATUS "Unix gconf probe: ${OPENSCAP_PROBE_UNIX_GCONF}")
+message(STATUS "Unix process58 probe: ${OPENSCAP_PROBE_UNIX_PROCESS58}")
+message(STATUS " ")
+
+message(STATUS "LINUX PROBES (depending of specific libs):")
+message(STATUS "Linux systemdunitproperty probe (depends of dbus): ${OPENSCAP_PROBE_LINUX_SYSTEMDUNITPROPERTY}")
+message(STATUS "Linux systemdunitdependency probe (depends of dbus): ${OPENSCAP_PROBE_LINUX_SYSTEMDUNITDEPENDENCY}")
```

Comment:
should be: depends on

Reply:
Good idea, i've do that, example of output:
I will pay more attention to my english :D  

```
-- CMake:
-- generator: Unix Makefiles
-- source directory: /home/dom/openscap
-- build directory: /home/dom/openscap/build
--  
-- Core features:
-- SCE: ON
--  
-- OVAL:
-- base probe support: ON
-- SEAP msgid bit-size: 32
-- 
-- Independent probes: ON
--   Independent family probe: ON
--   Independent system info probe: ON
--   Independent variable probe: ON
-- 
-- Independent probes incompatible with WIN32 (WIN32 status: FALSE)
--   Independent environmentvariable probe: ON
--   Independent environmentvariable58 probe: ON
--   Independent filehash probe: ON
--   Independent filehash58 probe: ON
--   Independent sql probe (depends on opendbx): OFF
--   Independent sql57 probe (depends on opendbx): OFF
--   Independent textfilecontent probe: ON
--   Independent textfilecontent54 probe: ON
--   Independent xmlfilecontent probe: ON
--  
-- Unix probes: ON
--   Unix dnscache probe: ON
--   Unix file probe: ON
--   Unix fileextendedattribute probe (depends on xattrh): OFF
--   Unix gconf probe (depends on gconf): ON
--   Unix interface probe: ON
--   Unix password probe: ON
--   Unix process probe: ON
--   Unix process58 probe (depends on CAP): ON
--   Unix routingtable probe: ON
--   Unix runlevel probe: ON
--   Unix shadow probe: ON
--   Unix symlink probe: ON
--   Unix sysctl probe: ON
--   Unix uname probe: ON
--   Unix xinetd probe: ON
--  
-- Linux probes: ON
--   Linux dpkginfo probe (depends on aptpkg): ON
--   Linux iflisteners probe: ON
--   Linux inetlisteningservers probe: ON
--   Linux partition probe (depends on blkid): ON
--   Linux rpminfo probe (depends on rpm): ON
--   Linux rpmverify probe (depends on rpm): ON
--   Linux rpmverifyfile probe (depends on rpm): ON
--   Linux rpmverifypackage probe (depends on rpm): ON
--   Linux selinuxboolean probe (depends on selinux): ON
--   Linux selinuxsecuritycontext probe (depends on selinux): ON
--   Linux systemdunitdependency probe (depends on dbus): ON
--   Linux systemdunitproperty probe (depends on dbus): ON
--  
-- Solaris probes: OFF
--   Solaris isainfo probe: OFF
--  
-- Windows probes: OFF
--   Windows accesstoken probe: OFF
--   Windows registry probe: OFF
--   Windows wmi57 probe: OFF
--  
-- Language bindings:
-- python3 bindings: ON
-- perl bindings: ON
[...]
```